assets/js/lib/operations/index.js

```diff
@@ -38,6 +43,8 @@ const OPERATION_LABELS = {
 const OPERATION_INTERNAL_NAMES = {
   'saptuneapplysolution@v1': SAPTUNE_SOLUTION_APPLY,
   'saptunechangesolution@v1': SAPTUNE_SOLUTION_CHANGE,
+  'clusterhoststart@v1': CLUSTER_HOST_START_OPERATION,
```

Comment:
This should be what we return from wanda, and I think the value is `crmclusterstart@v1`.
And the value in the map should be `CLUSTER_HOST_START`, without the `OPERATION`